pkg/metrics/metrics.go

```diff
@@ -28,17 +28,18 @@ type ServerMetrics struct {
 }
 
 const (
-	metricNamespace             = "ark"
-	backupTarballSizeBytesGauge = "backup_tarball_size_bytes"
-	// TODO: Rename the Count variables to match their strings
-	backupAttemptCount           = "backup_attempt_total"
-	backupSuccessCount           = "backup_success_total"
-	backupFailureCount           = "backup_failure_total"
-	backupDurationSeconds        = "backup_duration_seconds"
-	restoreAttemptTotal          = "restore_attempt_total"
-	restoreValidationFailedTotal = "restore_validation_failed_total"
-	restoreSuccessTotal          = "restore_success_total"
-	restoreFailedTotal           = "restore_failed_total"
+	metricNamespace               = "ark"
+	backupTarballSizeBytesGauge   = "backup_tarball_size_bytes"
+	backupAttemptTotal            = "backup_attempt_total"
+	backupSuccessTotal            = "backup_success_total"
+	backupFailureTotal            = "backup_failure_total"
+	backupDurationSeconds         = "backup_duration_seconds"
+	restoreAttemptTotal           = "restore_attempt_total"
+	restoreValidationFailedTotal  = "restore_validation_failed_total"
+	restoreSuccessTotal           = "restore_success_total"
+	restoreFailedTotal            = "restore_failed_total"
+	volumeSnapshotsAttemptTotal   = "volume_snapshot_attempt_total"
+	volumeSnapshotsCompletedTotal = "volume_snapshot_success_total"
```

Comment:
Let's call the variable `volumeSnapshotsSuccessTotal` to match the metric name. Please also add `volumeSnapshotsFailureTotal`.

Reply:
I believe a new field needs to be added to `BackupStatus` here for that and then we can check the status?
https://github.com/heptio/ark/blob/master/pkg/apis/ark/v1/backup.go#L163

Reply:
It's the difference between attempted & completed (right @skriss?)

Reply:
correct

Reply:
Oh, I added a new field for that and computed it based on `SnapshotPhaseFailed` - is that incorrect, @skriss?